src/sensors/distance/ultrasound/i2c/SRF08.cpp

```diff
@@ -125,8 +123,8 @@ uint8_t SRF08::getLightReading()
     mRuntime.i2cRequestFrom(mAddress, kNumberOfBytesToRequest);
     mRuntime.delayMillis(mPingDelay);
 
-    return mRuntime.i2cAvailable() != 0 ? static_cast<uint8_t>(mRuntime.i2cRead())
-                                        : static_cast<uint8_t>(-1);
+    return mRuntime.i2cAvailable() ? static_cast<uint8_t>(mRuntime.i2cRead())
```

Comment:
:warning: **readability-implicit-bool-conversion** :warning:
implicit conversion 'int' -> bool
```suggestion
    return  != 0```